rendering/glb.ksy

```diff
@@ -0,0 +1,64 @@
+meta:
+  id: glb
+  title: glTF binary format
+  file-extension: glb
+  endian: le
+  license: MIT
+  xref:
+    doc-ref: https://github.com/KhronosGroup/glTF/tree/master/specification/2.0#binary-gltf-layout
```

Comment:
Incorrect use of `doc-ref`. Feel free to merge your block with https://github.com/kaitai-io/kaitai_struct_formats/issues/209 Also add `Closes: #209` into PR text.

Reply:
Incorrect use? I originally had it one level up but the compiler gave an error that didn't tell me how to fix it, and then didn't complain when I moved it inside the refs section.

What's the right place?

I have to admit, I'm not used to working with doc-ref because I usually work on formats with zero documentation.

Reply:
What's the right place?

Not in `meta`, just under `doc`, wuich is a sibling of `meta`.

Reply:
Aha. It would have been nice for the compiler to say that, instead of just saying that it was wrong.

Reply:
> Is the compiler just not validating this section at all?

It is likely that in this section keys can be any strings.

Reply:
Sounds like a recipe for disaster...

In any case, it's fixed now.